widgets/filters/src/FiltersTable.tsx

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash';
 
-import type { Filter, Widget } from './types';
+import { FunctionComponent } from 'react';
```

Comment:
Optional:

Could we do

```suggestion
import type { FunctionComponent } from 'react';
```

?

To prevent situations like these, where we do not add `import type` manually, I have found out recently there's a setting in `tsconfig.json` we can enable that will mark these as errors. I will add a ticket in the backlog to enable that in `cloudify-ui-common`. That will let us avoid some review comments :slightly_smiling_face: 

The ticket: https://cloudifysource.atlassian.net/browse/RD-1862